aws-crt-kotlin/common/test/aws/sdk/kotlin/crt/http/HttpClientConnectionTest.kt

```diff
@@ -124,7 +124,6 @@ class HttpClientConnectionTest : CrtTest() {
      * Connect to the URL with all TLS ciphers supported. Throws an exception if the connection attempt fails for
      * any reason
      */
-    @OptIn(ExperimentalTime::class)
     private suspend fun connectAllCiphers(url: String) {
```

Comment:
**Question**: Did something (IDE, Gradle, etc.) proactively warn you about the unnecessity of this opt-in? Or did you just happen to see this and elect to remove it?

Reply:
The IDE warned me. I did a global code search for all `@OptIn` annotations, and removed the ones which the IDE deemed unnecessary